### DISCUSSION TITLE
Is building jellyfin-tizen with this repo possible

### DISCUSSION THREAD

**Discussion by HEPOSHEIKKI** [jellyfin/jellyfin-vue#1891] [Questions] [answered]
Hi!
I tried building jellyfin-tizen with the production build of this repo, but it produces an unfunctional wgt. This is not a surprise as this repo is not meant to be used with jellyfin-tizen afaik. Is there any way to build jellyfin tizen using this repo now or in the future or are jellyfin-web and jellyfin-vue so fundamentally different it just isn't possible?

**Comment by ferferga** [comment, marked as answer]
I guess you can try yourself, I don't own an LG TV. Probably it will only work in recent models, since we're using the latest browser features without any polyfills (backward compatibility)

Maybe the reason it doesn't work it's because your TV its too old, but in recent models it might work.

Let's see if somebody else tries!
